simapp/helpers/service_msg_client.go

```diff
@@ -0,0 +1,50 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+
+	gogogrpc "github.com/gogo/protobuf/grpc"
+	grpc "google.golang.org/grpc"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ gogogrpc.ClientConn = &ServiceMsgClientConn{}
+
+// ServiceMsgClientConn is an instance of grpc.ClientConn that is used to test building
+// transactions with MsgClient's. It is intended to be replaced by the work in
+// https://github.com/cosmos/cosmos-sdk/issues/7541 when that is ready.
+type ServiceMsgClientConn struct {
```

Comment:
If we merge this PR, this struct is going to be defined three times:
- once here
- once in x/feegrant/cli
- once in x/bank/client/testutil

It would be ideal to only define it once.

Maybe in `types/msgservice` for now? Alternatively, we could merge https://github.com/cosmos/cosmos-sdk/pull/7630.

Reply:
This one is used only in simulations. So, how about putting it in a package for tests / simulations?

Reply:
No, it's needed in CLI, which is too "important" imo to put in a test/helper/simulation folder.